api/posters/statistics/helpers.ts

```diff
@@ -55,28 +56,37 @@ const getPerLanguageStatiticsContent = (
   stats: PerLanguageStats,
   totalPosters: number,
 ) => {
-  const rows = Object.entries(stats).map(
-    ([language, total]) => `<tr><td>${language}</td><td>${total}</td></tr>`,
-  )
+  const perLanguageHtmlRows: string = Object.entries(stats)
+    .map(
+      ([language, total]) =>
+        `<tr style='${bordered}'>
+          <td style='${bordered} ${paddedContent} min-width: 75px;'>
+            <em>${language}</em>
+          </td>
+          <td style='text-align: center; min-width: 50px; ${paddedContent}'>
+            <strong>${total}</strong>
+          </td>
+        </tr>`,
+    )
+    .join('')
```

Comment:
I know we have already used vanilla HTML to render this. However, in the future, if you need more customization feel free to use React and server render it with ReactDOM. There are some snippets in the `start.utils.ts` that do this. 

It's not a blocker, but if we need better styles perhaps we could use it.